kratos/tests/cpp_tests/sources/test_model_part.cpp

```diff
@@ -55,6 +55,19 @@ namespace Kratos {
         rModelPart.CreateNewGeometry("Triangle2D3", "Geometry_8", {{5,6,3}});
     }
 
+    KRATOS_TEST_CASE_IN_SUITE(ModelPartDataValueContainerAndFlag, KratosCoreFastSuite)
+    {
+        Model model;
+        ModelPart& r_model_part = model.CreateModelPart("Main");
+        r_model_part.SetValue(DENSITY,1.0);
+        KRATOS_CHECK(r_model_part.Has(DENSITY));
+        KRATOS_CHECK_IS_FALSE(r_model_part.Has(TEMPERATURE));
+        KRATOS_CHECK_DOUBLE_EQUAL(r_model_part.GetValue(DENSITY),1.0);
+        r_model_part.Set(ACTIVE);
+        KRATOS_CHECK(r_model_part.Is(ACTIVE));
+        KRATOS_CHECK_IS_FALSE(r_model_part.Is(BOUNDARY));
+    }
+
```

Comment:
```suggestion
    KRATOS_TEST_CASE_IN_SUITE(ModelPartDataValueContainer, KratosCoreFastSuite)
    {
        Model model;
        ModelPart& r_model_part = model.CreateModelPart("Main");
        r_model_part.SetValue(DENSITY,1.0);
        KRATOS_CHECK(r_model_part.Has(DENSITY));
        KRATOS_CHECK_IS_FALSE(r_model_part.Has(TEMPERATURE));
        KRATOS_CHECK_DOUBLE_EQUAL(r_model_part.GetValue(DENSITY),1.0);
    }

    KRATOS_TEST_CASE_IN_SUITE(ModelPartFlag, KratosCoreFastSuite)
    {
        r_model_part.Set(ACTIVE);
        KRATOS_CHECK(r_model_part.Is(ACTIVE));
        KRATOS_CHECK_IS_FALSE(r_model_part.Is(BOUNDARY));
    }

```
Admittedly a bit picky but I'd separate the test in two as you did for the Python ones.

Reply:
I will as an independent PR, if I need to trigger the CI anyway this will simplify review and speed up process